.tests/integration/config.yaml

```diff
@@ -26,6 +26,20 @@ cnvkit_scatter:
 cnvkit_vcf:
   container: "docker://hydragenetics/cnvkit:0.9.9"
 
+cnvkit_seg:
+  container: "docker://hydragenetics/cnvkit:0.9.9"
+
+purecn:
+  container: "docker://markusriester/purecn:2.2.0"
+  genome: "hg19"
+  segmentation_method: "GATK4"
+  fun_segmentation: "Hclust"
+  interval_padding: 100
+
+purecn_coverage:
+  container: "docker://markusriester/purecn:2.2.0"
```

Comment:
would it be possible to build a container and upload it to hydra-genetics organization? ? 

can be done at: https://github.com/hydra-genetics/docker

Reply:
Yes, although the [Dockerfile for markusriester/purecn](https://github.com/lima1/PureCN/blob/master/Dockerfile) is a bit complicated to build support for GenomicsDB. So in that case it would probably just be `FROM markusriester/purecn:2.2.0` without much more added. Is that OK?

Reply:
That's ok for me! At least as a start, in the future we could look at having a complete build file (should make it easier to update version)

Reply:
I created a PR for this (hydra-genetics/docker#95)